src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -2030,6 +2030,165 @@ public async Task ValidateErrorMessageForMutationWithoutReadPermission()
+                                                                                clientRoleHeader: null);
+
+                Assert.IsNotNull(mutationResponse);
+
+                SqlTestHelper.TestForErrorInGraphQLResponse(
+                                mutationResponse.ToString(),
+                                message: "The specified input object field `publishers` does not exist.",
+                                path: @"[""createbook""]");
+
+                string manyTypeMultipleCreateOperation = @"mutation {
+                                                              createbooks(
+                                                                items: [
+                                                                  { title: ""Book #1"", publishers: { name: ""Publisher #1"" } }
+                                                                  { title: ""Book #2"", publisher_id: 1234 }
+                                                                ]
+                                                              ) {
+                                                                items {
+                                                                  id
+                                                                  title
+                                                                }
+                                                              }
+                                                            }";
+
+                mutationResponse = await GraphQLRequestExecutor.PostGraphQLRequestAsync(client,
+                                                                                        server.Services.GetRequiredService<RuntimeConfigProvider>(),
+                                                                                        query: manyTypeMultipleCreateOperation,
+                                                                                        queryName: "createbook",
+                                                                                        variables: null,
+                                                                                        clientRoleHeader: null);
+
+                Assert.IsNotNull(mutationResponse);
+                SqlTestHelper.TestForErrorInGraphQLResponse(mutationResponse.ToString(),
+                                                            message: "The field `createbooks` does not exist on the type `Mutation`.");
+
+                string pointCreateOperation = @"mutation createbook{
+                                                            createbook(item: { title: ""Book #1"", publisher_id: 1234 }) {
+                                                                title
+                                                                publisher_id
+                                                            }
+                                                        }";
```

Comment:
Thanks for the suggestion.

I've added two additional tests: 
1. Point create mutation with missing `publisher_id` field with multiple create operation disabled. This should be caught by hotchocolate since a required field is missing. 
2. Point create mutation with missing `publisher_id` field with multiple create operation enabled. This should also lead to a failure. But, the error gets thrown from the database layer since there is no sufficient data to successfully execute the insert statement.  
Note: Once request validation PR gets merged, these types of requests will fail at the request validation stage (the flow will not reach till the database layer). @ayush3797, the error message that is checked in the test will have to be updated when checking-in the request validation PR.